services/github/src/handlers/status.js

```diff
@@ -70,6 +70,14 @@ export async function statusHandler(message) {
   }
 
   const { organization, repository, sha, event_id, event_type, installation_id } = build;
+
+  // Validate required build properties
+  if (!sha || !organization || !repository) {
```

Comment:
This doesn't make any sense -  db Schema for `github_builds` doesn't allow null values on any of those fields